### PR TITLE
Added getTech and evalTclString to Design

### PR DESCRIPTION
**Pull request by macd**
Signed-off-by: Don MacMillen <don@macmillen.net>

### CODE REVIEW

**Review by maliberty — APPROVED** [review]
Remember to request a review

**Comment by macd** [comment]
Yes, I was looking at that but there was no indication on how to do that. Looking at the github docs it looks like only owners and collaborators can request a review.

File: include/ord/Design.h [hunk 1]
```diff
@@ -73,6 +73,10 @@ class Design
 
   int micronToDBU(double coord);
 
+  int evalTclString(const std::string& cmd);
```
Comment (maliberty) [inline include/ord/Design.h:76]: Please add a comment that this is a temporary backdoor and not a permanent feature.

**Review by maliberty — APPROVED** [review]

File: include/ord/Design.h (hunk 1 above)
Comment (macd) [reply include/ord/Design.h:77]: Will do. (CI in progress)

**Pull request merged**